fixes this chromium warning which turns into error:

________ running 'download_from_google_storage --no_resume --platform=linux* --no_auth --bucket chromium-gn -s src/buildtools/linux32/gn.sha1' in '/home/emacs/vmb/build/chromium-dev-git/fetch_gclient_base/checkout_root'
/opt/depot-tools-git/third_party/boto/pyami/config.py:75: UserWarning: Unable to load AWS_CREDENTIAL_FILE ()
  warnings.warn('Unable to load AWS_CREDENTIAL_FILE (%s)' % full_path)
Copying gs://chromium-gn/26db2ae201b26761164a93c3ada56280a68539ec...

0> Downloading src/buildtools/linux32/gn...
0> /opt/depot-tools-git/third_party/boto/pyami/config.py:75: UserWarning: Unable to load AWS_CREDENTIAL_FILE ()
  warnings.warn('Unable to load AWS_CREDENTIAL_FILE (%s)' % full_path)
Copying gs://chromium-gn/26db2ae201b26761164a93c3ada56280a68539ec...

Error: Command download_from_google_storage --no_resume --platform=linux* --no_auth --bucket chromium-gn -s src/buildtools/linux32/gn.sha1 returned non-zero exit status 1 in /home/emacs/vmb/build/chromium-dev-git/fetch_gclient_base/checkout_root


diff --git a/third_party/boto/pyami/config.py b/third_party/boto/pyami/config.py
index 08da658..9d948a1 100644
--- a/third_party/boto/pyami/config.py
+++ b/third_party/boto/pyami/config.py
@@ -67,7 +67,7 @@ class Config(ConfigParser.SafeConfigParser):
                 self.readfp(fp)
             else:
                 self.read(BotoConfigLocations)
-            if "AWS_CREDENTIAL_FILE" in os.environ:
+            if "AWS_CREDENTIAL_FILE" in os.environ and os.environ['AWS_CREDENTIAL_FILE'].strip() != "":
                 full_path = expanduser(os.environ['AWS_CREDENTIAL_FILE'])
                 try:
                     self.load_credential_file(full_path)
